tests/system/action/motion/test_create.py

```diff
@@ -409,3 +409,1358 @@ def test_create_with_submitters(self) -> None:
+
+    def test_create_set_number_return_because_number_preset(self) -> None:
+        self.create_model(
+            "meeting/222", {"name": "name_SNLGsvIV", "motions_number_type": "manually"}
+        )
+        self.create_model(
+            "motion_workflow/12",
+            {
+                "name": "name_workflow1",
+                "first_state_id": 34,
+                "state_ids": [34],
+            },
+        )
+        self.create_model(
+            "motion_state/34",
+            {"name": "name_state34", "meeting_id": 222, "set_number": True},
+        )
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion.create",
+                    "data": [
+                        {
+                            "title": "test_Xcdfgee",
+                            "meeting_id": 222,
+                            "workflow_id": 12,
+                            "text": "test",
+                            "number": "A003",
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("motion/1")
+        assert model.get("title") == "test_Xcdfgee"
+        assert model.get("meeting_id") == 222
+        assert model.get("number") == "A003"
+
+    def test_create_set_number_return_because_number_type_manually(self) -> None:
+        self.create_model(
+            "meeting/222", {"name": "name_SNLGsvIV", "motions_number_type": "manually"}
+        )
+        self.create_model(
+            "motion_workflow/12",
+            {
+                "name": "name_workflow1",
+                "first_state_id": 34,
+                "state_ids": [34],
+            },
+        )
+        self.create_model(
+            "motion_state/34",
+            {"name": "name_state34", "meeting_id": 222, "set_number": True},
+        )
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "motion.create",
+                    "data": [
+                        {
+                            "title": "test_Xcdfgee",
+                            "meeting_id": 222,
+                            "workflow_id": 12,
+                            "agenda_create": True,
+                            "text": "test",
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("motion/1")
+        assert model.get("title") == "test_Xcdfgee"
+        assert model.get("meeting_id") == 222
+        assert model.get("number") is None
+        agenda_item = self.get_model("agenda_item/1")
+        self.assertEqual(agenda_item.get("meeting_id"), 222)
+        self.assertEqual(agenda_item.get("content_object_id"), "motion/1")
+
+    def test_create_set_number_return_because_no_set_number(self) -> None:
```

Comment:
this test does the same as the test above, but does not create an agenda item.

Reply:
Removed the double test.